spec/kokoro-run-all.sh

```diff
@@ -197,4 +196,7 @@ else
   done
 fi
 
+chmod +x $KOKORO_GFILE_DIR/linux_amd64/buildcop
```

Comment:
Think I saw this in the E2E block, so wanted to comment here for when you go to move it back. I think E2E tests only run on one version, and possibly on PRs. So, we should do:

```
if [[ $KOKORO_BUILD_ARTIFACTS_SUBDIR = *"system-tests"* ]]; then
  ...
fi
```